js/react_native/lib/index.ts

```diff
@@ -3,9 +3,16 @@
 
 export * from 'onnxruntime-common';
 import {registerBackend, env} from 'onnxruntime-common';
+import {Platform} from 'react-native';
 import {onnxruntimeBackend} from './backend';
 import {version} from './version';
 
 registerBackend('cpu', onnxruntimeBackend, 1);
+registerBackend('xnnpack', onnxruntimeBackend, 1);
+if (Platform.OS === 'android') {
+  registerBackend('nnapi', onnxruntimeBackend, 1);
+} else if (Platform.OS === 'ios') {
+  registerBackend('coreml', onnxruntimeBackend, 1);
+}
```

Comment:
is there a way to query whether a specific EP is supported or not in the current build?

I am not sure whether the default build supports those EPs (NNAPI for android and CoreML for iOS), unless they are always available ( need verify @skottmckay), it's better to do a runtime check here to make sure they are available before registering them.

Reply:
I think currently it should can be check in native module build of React Native, it should fail once onnxruntime-c / onnxruntime-java does not support these EPs.

Reply:
They're available in the mobile packages the ORT RN setup uses. 

AFAIK there's no consistent way to do a runtime check of which EPs are available due to how some of the registration was setup originally in the native code.